.github/workflows/main.yml

```diff
@@ -39,6 +39,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: [yarn_lock_check]
     env:
+      NX_PARALLEL: 3
```

Comment:
Can't we set that directly in GitHub settings?

Reply:
We can but it will be apply to all nx command, not sure it is what we want.
I saw a time degradation for `build` task but not for `Unit test` which seems to be faster

Reply:
do we really want to have a different value depending on the job? I think we should have the same value everywhere

Reply:
If we decide to not have the same value everywhere I would expect a comment to explain why do we not use the default one